diff -x .git -ruN teste1/src/app/api/fit/route.ts /home/ofe/.local/teste/src/app/api/fit/route.ts
--- teste1/src/app/api/fit/route.ts	2025-03-05 20:22:45.602130051 +0000
+++ /home/ofe/.local/teste/src/app/api/fit/route.ts	2025-03-05 20:21:19.004347993 +0000
@@ -120,7 +120,7 @@
     form.append("file", jsonBlob, "data.json");
 
     const response = await fetch(
-      "http://onefite-t.vps.tecnico.ulisboa.pt:8142/fit",
+      "http://127.0.0.1:8142/fit",
       {
         method: "POST",
         body: form as any,
@@ -208,7 +208,7 @@
     Num: 500,
     Dados: `# DATA ${selectedDataset.auxiliarIndependentVariablesArrayName} = ${selectedDataset.auxiliarIndependentVariablesArray.join(" ")}\n# TAG = tags.txt\n${dataString}`,
     FitType: isFitGlobal ? "Global" : "Individual",
-    Function: independentVariable + " = " + processedFunction,
+    Function: dependentVariable + " = " + processedFunction,
     Parameters: parameters.map((param) => param.name).join(","),
     X: independentVariable,
     Xmax: (maxIndependentVariable + independentVariableMargin).toString(),
@@ -329,7 +329,7 @@
     Num: 500,
     Dados: dados,
     FitType: isFitGlobal ? "Global" : "Individual",
-    Function: independentVariable + " = " + processedFunction,
+    Function: dependentVariable + " = " + processedFunction,
     Parameters: parameters.map((param) => param.name).join(","),
     X: independentVariable,
     // For each entry in the
